ui/lib/core/addon/templates/components/replication-table-rows.hbs

```diff
@@ -2,13 +2,26 @@
   <EmptyState />
 {{else}}
   <div class="has-large-margin-top">
-    {{info-table-row label="Merkle root index" helperText="A snapshot in time of the merkle tree's root hash. Changes on every update to storage." value=merkleRoot}}
-    {{info-table-row label="Mode" value=mode}}
-    {{info-table-row label="Replication set" helperText="The ID for the group of clusters communicating for this replication mode" value=clusterId}}
+    {{info-table-row
```

Comment:
🙏

Reply:
Did you do this by hand or did you figure out how to get hbs files formatted automatically?!

Reply:
i did this by hand because it was hurting my eyes 🔥